release-notes.txt

```diff
@@ -1,3 +1,543 @@
+#8746: stm32_common/periph/rtc: current implementation broken/poor accuracy
+#8251: MSP430: periph_timer clock config wrong
+#7347: xtimer_usleep stuck for small values
+#7114: xtimer: add's items to the wrong list if the timer overflows between
+       _xtimer_now()  and irq_disable()
+#6442: cpu/native: timer interrupt issue
+#6052: tests: xtimer_drift gets stuck on native
+
+Drivers related issues (14)
+---------------------------
+#15121: PWM: Single-phase initialization creates flicker
+#14506: mdt_erase success, but vfs_format resets board (esp32-heltec-lora32-v2)
+#14439: (almost solved) SPI SD-Card driver: SPI initialisation freeze until
+        timeout
+#14424: adc is not a ADC-Driver but a analog pin abstraction
+#13527: examples/dtls-wolfssl not working on pba-d-01-kw2x
+#13444: Potential security and safety race conditions on attached devices
+#13079: drivers/srf04: incorrect values on ATmega based platforms
+#12445: driver/hts221: Temperature and Humidity readings incorrect
+#12371: fail to send data to can bus
+#12057: ESP32 + DHT + SAUL reading two endpoints causes freeze.
+#12045: floats and doubles being used all over the place.
+#11104: STM32: SPI clock not returning to idle state and generating additional
+        clock cycles
+#9419: cpu/msp430: GPIO driver doesn't work properly
+#4876: at86rf2xx: Simultaneous use of different transceiver types is not
+       supported
+
+Native related issues (3)
+-------------------------
+#13501: SDL2 does not work due to missing getpid
+#5796: native: tlsf: early malloc will lead to a crash
+#495: native not float safe
+
+Other platforms related issues (20)
+-----------------------------------
+#14822: MPU doesn't work on cortex-m0+
+#14728: stm32f7: Large performance difference between stm32f746 and stm32f767
+#14679: examples/gnrc_border_router: esp_wifi crashes on disconnect
+#14572: tests/mpu_noexec_ram: fails on  i-nucleo-lrwan1
+#14410: MIPS: toolchain objcopy doesn't work and no .bin can be generated
+#14237: esp32-wroom-32: tests/netstats_l2 failing sometimes
+#14015: stm32152re: hardfault when DBGMCU_CR_DBG* bits are set and branch after
+        __WFI()
+#13606: gcoap/esp8266: Stack overflow with gcoap example
+#13390: Cannot use LLVM with Cortex-M boards
+#13104: boards/hifive1: flashing issue
+#13086:  Failing tests on FE310 (Hifive1b)
+#12763: [TRACKING] Fixes for automatic tests of ESP32 boards.
+#12651: Failing tests on AVR (tested with atmega256rfr2-xpro)
+#12168: pkg/libb2: blake2s doesn't work on AVR
+#11885: arm7: printf() with float/double not working
+#10258: Incorrect defaultPORT building for esp32-wroom-32 on macOS
+#10122: Indeterministic hard fault in _mutex_lock(), with nRF52 SoftDevice
+#10076: cpu/cortexm_common: irq_enable returns the current state of interrupts
+        (not previous)
+#6567: periph/spi: Switching between CPOL=0,1 problems on Kinetis with software
+       CS
+#4954: chronos: compiling with -O0 breaks
```

Comment:
I think this issue no longer applies (see https://github.com/RIOT-OS/RIOT/issues/4954#issuecomment-718944290)

Reply:
Removed the issue and adjusted the counters